k2/csrc/fsa.h

```diff
@@ -13,9 +13,9 @@
 #include <utility>
 #include <vector>
 
-#include "glog/logging.h"
 #include "k2/csrc/array.h"
 #include "k2/csrc/util.h"
+#include "k2/csrc/util/logging.h"
```

Comment:
I think you were going to get rid of the directory util/?

Reply:
mm.. there is more code than I realized, now wondering whether we should keep util/

Reply:
I expect more subdirs would be here, as k2 surely needs more function ((IDK) IO, bin ..) that should distribute more organized. 

And I think the dlpack related codes under `python/csrc/` may be better under `csrc/dlpack` as dlpack is not for pybind essentially (dlpack gives a way to transfer between data structures, as here we use it transfer k2.Tensor with torch.Tensor). Pytorch put it as `from torch.utils.dlpack import to_dlpack` in py API. I think we would need the class k2.Tensor from current `python/csrc/tensor.h`. 

Anyway, I hold no opinion about the right project structure🤗. Changes happen fast. So, `util/` is OK for now?